Cargo.toml

```diff
@@ -27,7 +29,7 @@ thiserror = "1.0.30"
 [dev-dependencies]
 anyhow = "1.0.66"
 clap = { version = "4.0.29", features = ["derive"] }
-rcgen = "0.12"
+rcgen = "0.12.1"
```

Comment:
This is unnecessary -- Cargo uses `^` for dependencies implicitly.

Reply:
Sorry a leftover after attempting to upgrade it and running into multiple breaking changes, I'll drop the `.1`!

Reply:
Curious though why the patch version is left off from this one, but all the other dependencies include it?

Reply:
Typically when you first add a dependency you take the current version, and then when you do a semver-incompatible bump you leave off the patch version (because `.0` patch versions are meaningless).